### ISSUE TITLE
xsd validate a semantic version but not powershell

### ISSUE THREAD

**Issue by LaurentDardenne** [PowerShellOrg/Plaster#238]
This version (A.0.0) is invalid for the xsd and for Powershell 5.1 :
```xml
  <requireModule name="Pester" condition='$PLASTER_PARAM_Options -contains "Pester"' minimumVersion="A.0.0"
     message="Without Pester ..."/>
```
> WARNING: Failed to create dynamic parameters from the template's manifest file.  Template-based dynamic parameters will not be available until the error is corrected.  The error was: The Plaster manifest
> 'G:\PS\Plaster\TestProject\plasterManifest.xml' is not valid. Specify -Verbose to see the specific schema errors.
> #... 
> 
> Plaster\Test-PlasterManifest : The Plaster manifest 'G:\PS\Plaster\TestProject\plasterManifest.xml' is not valid.  Specify -Verbose to see the specific schema errors.

But this version (1.0.0-beta.2) is valid for the xsd but not for Powershell 5.1 :

```xml
<requireModule name="Pester" condition='$PLASTER_PARAM_Options -contains "Pester"' minimumVersion="1.0.0-beta.2"
   message="Without Pester ..."/>
```

> invoke-plaster : Cannot bind parameter 'FullyQualifiedName'. Cannot convert value "System.Collections.Hashtable" to type "Microsoft.PowerShell.Commands.ModuleSpecification". Error: "Cannot convert value "1.0.0-beta.2" to type "System.Version". Error: "Le format de la chaîne d'entrée est incorrect.""

**Comment by rkeithhill** [comment]
That version attributes on the `RequireModule` directive map to the Version part of the `-FullyQualifiedName` parameter. That value has to be a valid `System.Version` string.  `A.0.0` is not a valid System.Version.  I probably should not have weaseled in support for semantic versioning prelease/build metadata but at the time I thought it would be coming.  Doesn't look that way know so I may need to rethink that regex and pull the support for prerelease/build metadata.

**Comment by jdhitsolutions** [comment]
closed as stale and hopefully fixed in v1.1.0. If not, we can re-open.
